### DISCUSSION TITLE
Does K1C is supported by K1 script?

### DISCUSSION THREAD

**Discussion by leozzhol** [Guilouz/Creality-Helper-Script-Wiki#703] [General]
Hi,
I have lots of experience with 3d printing, but i´ve never had a Klipper firmware one. I´ve just bought a K1C and everyone tells me to get everyting the printer can provide to me, i need to root it to use klipper.
Does this script will help me with this? Is the K1 files compatible with my K1C, there is any consideration i have to be aware?

Sorry about my newbe questions.
Regards from Brazil
Leonardo

**Comment by jglerner** [comment]
TLDR: Yes you can. Just follow the WIki Instructions.

Leonardo, também tenho uma K1C e fiz o root da máquina e instalei o Klipper que dá para colocar que é algo limitado.
Funciona 100%, nenhum problema. Depois, quando você aprender não vai querer voltar para o anterior de nenhum jeito. 
Conte comigo para ajudá-lo em alguma dúvida.

Jacques
